python/ray/experimental/sgd/pytorch/examples/dcgan.py

```diff
@@ -238,8 +238,8 @@ def train_example(num_replicas=1, use_gpu=False, test_mode=False):
         use_gpu=use_gpu,
         batch_size=16 if test_mode else 512,
         backend="nccl" if use_gpu else "gloo")
-    for i in range(5):
-        stats = trainer.train()
+    for i in range(10):
+        stats = trainer.train(retries=3)
```

Comment:
nit: would prefer `retries` -> `max_retries` for clarity and to match the Ray API